modules/300-prometheus/monitoring/prometheus-rules/deprecation.yaml

```diff
@@ -30,3 +30,79 @@
           __meta_kubernetes_endpoint_address_target_kind -> __meta_kubernetes_endpointslice_address_target_kind
           __meta_kubernetes_endpoint_address_target_name -> __meta_kubernetes_endpointslice_address_target_name
           ```
+
+- name: d8.grafana.interval_deprecation
+  rules:
+    - alert: GrafanaDashboardPanelIntervalDeprecated
+      expr: |
+        d8_grafana_dashboards_deprecated_interval > 0
```

Comment:
```suggestion
        sum by (dashboard, panel, interval) (d8_grafana_dashboards_deprecated_interval) > 0
```

Reply:
Fixed + added message about query evaluation to the alert description